src/server/appl-device/services/WDARunner.ts

```diff
@@ -461,22 +461,38 @@ export class WdaRunner extends TypedEmitter<WdaRunnerEvents> {
             }
             this.wdaEventInAction = true;
             this.emit('status-change', { status: WdaStatus.IN_ACTION, text: '제어 중' });
-            // eslint-disable-next-line @typescript-eslint/ban-types
-            try {
+
+            for (let ii = 0; ii < 3; ii++) {
                 // eslint-disable-next-line @typescript-eslint/ban-types
-                await (<Function>ev)(driver);
-                this.emit('status-change', { status: WdaStatus.END_ACTION, text: '제어 완료' });
-            } catch (e) {
-                this.logger.error(e);
-                this.emit('status-change', {
-                    status: WdaStatus.END_ACTION,
-                    text: '제어 실패',
-                    error: e.stack || e.message || e,
-                });
-            } finally {
-                this.wdaEventInAction = false;
-                this.emit('status-change', { status: WdaStatus.END_ACTION, text: '제어 완료' });
+                try {
+                    // eslint-disable-next-line @typescript-eslint/ban-types
+                    await (<Function>ev)(driver);
+                    this.emit('status-change', { status: WdaStatus.END_ACTION, text: '제어 완료' });
```

Comment:
@metocks // 1회 라도 성공하면 루프를 빠져나가야 하지 않을까요?
```suggestion
                    this.emit('status-change', { status: WdaStatus.END_ACTION, text: '제어 완료' });
                    break;
```